nixos/modules/programs/neovim.nix

```diff
@@ -75,6 +75,9 @@ in
           customRC = '''
             " here your custom configuration goes!
           ''';
+          customLuaRC = '''
+            -- or if you prefer Lua, your Lua configuration goes here!
```

Comment:
This comment doesn't make sense if you consume it without `customRC` above it. I think it'd be good to make it more 'standalone' for search.nixos.org et al